eventlet/tpool.py

```diff
@@ -282,18 +282,23 @@ def setup():
     _rsock = greenio.GreenSocket(csock)
     _rsock.settimeout(None)
 
-    for i in six.moves.range(_nthreads):
-        t = threading.Thread(target=tworker,
-                             name="tpool_thread_%s" % i)
-        t.setDaemon(True)
-        t.start()
-        _threads.append(t)
+    # Create initial `_nthreads` threads
+    _spawn_threads()
 
     _coro = greenthread.spawn_n(tpool_trampoline)
     # This yield fixes subtle error with GreenSocket.__del__
     eventlet.sleep(0)
 
 
+def _spawn_threads():
+    for i in six.moves.range(len(_threads), _nthreads):
+        t = threading.Thread(target=tworker,
+                             name="tpool_thread_%s" % (i + 1))
```

Comment:
Please make it `t = threading.Thread(target=tworker, name='tpool_thread_{0}'.format(i + 1))`
I know it is not related to work, but since you touched this line, might as well remove the bad and ugly formatting.

Reply:
Done